src/components/plot/chart-with-effect.tsx

```diff
@@ -168,7 +170,7 @@ const ChartWithEffect = ({ config }: ChartWithEffectProps): JSX.Element => {
     }
   }
   return (
-    <div className='chart-container'>
+    <div className='chart-container' style={{maxWidth: appStyles?.max_width, maxHeight: appStyles?.max_height}}>
```

Comment:
These properties were intended to be for the whole app, not each specific chart in react grid layout. We should have them on the html element with class `plot-page` in `plot-control-grid.tsx`. This way the width we defineon `grid-container` will adjust based on a set `max-width`